### ISSUE TITLE
Xroad 6.19.0 binaries

### ISSUE THREAD

**Issue by kmpsaravanan** [nordic-institute/X-Road-development#11]
Dear NIIS,

We have set up a local Ubuntu repository and have mirrored NIIS repository to install 6.19.0 in one of the Ubuntu 14.0. We are facing issue with 6 deb files with the error "Hash Sum mismatch". While mirroring/downloading the debs from NIIS repo the connection got terminated intermittently. After many tries, we copied the debs to our local repository, but we are getting the above hash error while installing the security server.

Below is the screen shot of the error

<img width="726" alt="screenshot 2018-12-26 at 09 54 37" src="https://user-images.githubusercontent.com/15813330/50431919-5c97b780-08f4-11e9-8f00-8a5438433a09.png">

Thank you,
Saro

**Comment by petkivim** [comment]
Hi @kmpsaravanan 

Could you try the installation using the repository below? Please let me know if you're able to complete the installation using this repository.

```
sudo apt-add-repository -y "deb https://xroad-release-mirror1.s3.amazonaws.com/xroad-release-deb $(lsb_release -sc)-current main"
```

Regards,
Petteri

**Comment by kmpsaravanan** [comment]
Thank you, this aws mirror link is working. We are able to install 6.19.

Thank you for your help, however kindly inform the NIIS team to fix the artificatory repo permanently.

**Comment by kmpsaravanan** [comment]
Dear NiiS team,

We ran performance test using xroad between two Information systems. We would like to know if we can capture the throughput between client security server and producer security server. We want capture the time it took for these two security server to process the API request and responses between the two Information systems.

Any reports available through xroad database or logs that can we can gather these details.

Regards,
Saro

Get Outlook for Android<https://aka.ms/ghei36>

**Comment by petkivim** [comment]
Hi @kmpsaravanan,

Operational monitoring provides you the information you need. The documentation can be found at:

* https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#15-operational-monitoring
* https://github.com/nordic-institute/X-Road/tree/develop/doc/OperationalMonitoring

Please be aware that in the versions <= 6.19.0 there’s an error in measuring the processing time of requests - time that is consumed between sending out a request and receiving a response. The logic may cause operational monitoring to return incorrect and even negative values. The error is fixed in version 6.20.0 that will be released after few weeks. More information about the error can be found at:

https://jira.niis.org/browse/XRDDEV-108

Regards,
Petteri
